mavros_msgs/msg/CompanionProcessStatus.msg

```diff
@@ -0,0 +1,59 @@
+# Mavros message: COMPANIONPROCESSSTATUS
+
+std_msgs/Header header
+
+uint8 state        		# See enum COMPANION_PROCESS_STATE
+uint8 component         # See enum MAV_COMPONENT
+
+uint8 MAV_STATE_UNINIT = 0
+uint8 MAV_STATE_BOOT = 1
+uint8 MAV_STATE_CALIBRATING = 2
+uint8 MAV_STATE_STANDBY = 3
+uint8 MAV_STATE_ACTIVE = 4
+uint8 MAV_STATE_CRITICAL = 5
+uint8 MAV_STATE_EMERGENCY = 6
+uint8 MAV_STATE_POWEROFF = 7
+uint8 MAV_STATE_FLIGHT_TERMINATION = 8
+
+uint8 MAV_COMP_ID_ALL = 0
```

Comment:
@baumanta we don't need all of these enumeration entries, since the only `MAV_COMP`  we require are the ones used for the companion processes.